ApiReview/API.Polly.Core/NoDocs/Polly.Retry/RetryStrategyOptions.cs

```diff
@@ -0,0 +1,25 @@
+// Assembly 'Polly.Core'
+
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.Runtime.CompilerServices;
+using System.Threading.Tasks;
+using Polly.Strategy;
+
+namespace Polly.Retry;
+
+public class RetryStrategyOptions : ResilienceStrategyOptions
+{
+    public const int InfiniteRetryCount = -1;
+    public sealed override string StrategyType { get; }
+    [Range(-1, 100)]
```

Comment:
If you support infinite retries, then why limit this to 100? I can have infinite, but I can't have 101 retries? I think the range should just be -1..int.MaxValue

Reply:
Change validation to 1 to int.maxvalue